src/contants/colors.ts

```diff
@@ -0,0 +1,2 @@
+export const blueBase = "#007CAD";
```

Comment:
The colors in this file are available from the @leafygreen-ui/palette npm module. What do you think about importing the colors from there instead of this file?

Reply:
good idea